docs/apache-airflow/core-concepts/dags.rst

```diff
@@ -0,0 +1,5 @@
+# DAGs
+
+## Tag limitations
+
+Each individual tag must be 100 characters or less. Tags longer than 100 characters will cause a validation error during DAG parsing with a clear error message indicating the offending tag and its length.
```

Comment:
i'm not even sure we really need to document this since users will find out when they hit the error at parse tiem